sdk/src/main/java/com/dailymotion/android/player/sdk/events/PlayerEvent.kt

```diff
@@ -0,0 +1,44 @@
+package com.dailymotion.android.player.sdk.events
+
+import com.dailymotion.android.player.sdk.PlayerWebView
+
+sealed class PlayerEvent(val name: String, open val payload: String?) {
+
+    data class GenericPlayerEvent internal constructor(override val payload: String?) : PlayerEvent("_generic_", payload)
```

Comment:
Do we really need these events to be data classes ? I'm a bit reluctant to add data classes to public APIs. See https://jakewharton.com/public-api-challenges-in-kotlin/